src/wizard/javascript/profiling-onboarding/javascript/1.install.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Browser (JS Self Profiling)
+doc_link: https://docs.sentry.io/platforms/javascript/profiling/
+support_level: beta
+type: framework
+---
+
+#### Install
+
+Because our browser profiling integration is built on top of the profiler exposed by the [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/), it's in beta and will likely only move out once the official spec progresses. There are some risks involved with using a beta package. We've made sure that the SDK gracefully degrades in unsupported environments.
+
+Please note, that since profiling API is currently only implemented in Chromium based browsers, the profiles collected will inherently be biased towards that demographic.
```

Comment:
We could also link to this status page which shows whether other browsers are adopting it: https://chromestatus.com/feature/5170190448852992